sdk/tests/conformance2/vertex_arrays/vertex-array-object.html

```diff
@@ -476,17 +476,22 @@
     // delete the color buffers AND the position buffer.
     gl.bindVertexArray(null);
     for (var ii = 0; ii < colorBuffers.length; ++ii) {
-      gl.deleteBuffer(colorBuffers[ii]);
-      gl.deleteBuffer(elementBuffers[ii]);
       gl.bindVertexArray(vaos[ii]);
-      var boundBuffer = gl.getVertexAttrib(1, gl.VERTEX_ATTRIB_ARRAY_BUFFER_BINDING);
+
+      gl.deleteBuffer(colorBuffers[ii]);
+      gl.deleteBuffer(positionBuffer);
```

Comment:
If the VAO is bound to the current context (via bindVertexArray), *then* the buffer is supposed to be automatically unbound from the VAO during DeleteBuffers. Otherwise, the DeleteBuffers call is supposed to act as an "unref" operation, but the buffer is supposed to remain alive until the VAO is deleted.

Firefox and Chrome are correctly reporting that the test fails with this pull request applied -- the pull request is incorrect, sorry.